src/seedu/addressbook/data/person/Address.java

```diff
@@ -40,6 +54,14 @@ public static boolean isValidAddress(String test) {
     public String toString() {
         return value;
     }
+    
+    private void putInPlace(String[] items){
+    	block = new AddressBlock(items[BLOCK_IDX]);
```

Comment:
Be careful, this code can break if the input address is not correct (i.e. you may not have all the necessary parts).